command/create_coin.go

```diff
@@ -0,0 +1,83 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hex337/alex-koin-go/model"
+
+	"log"
+	"regexp"
+)
+
+type CreateCoinCommand struct{}
+
+func (c *CreateCoinCommand) Run(msg string, event *CoinEvent) (string, error) {
+	re := regexp.MustCompile(`^(?i)create koin \<@(?P<to_slack_id>[0-9A-Z]+)\> (?:for)??(?P<reason>.+)`)
+	matches := re.FindStringSubmatch(event.Message)
```

Comment:
Note, probably need to figure out how to verify that we got the right number of matches here.